recipes/larch/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "larch" %}
+{% set version = "5.3.0" %}
+{% set checksum = "2a5d51ccaf05a31c7c0bbef17f4428f72c7dbd673b259776896136d03108b9ac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jpn--/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [py<37]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - compilers
```

Comment:
the `compilers` package should not be used in recipes.  It is only for use by end-users.  please use the `{{ compiler() }}`  calls

Reply:
Removed the “compilers” dependency.